tests/share/test_packager.py

```diff
@@ -819,23 +820,27 @@ def test_view_over_snapshot_round_trip(self, reset_db: None) -> None:
         for i in (3, 0, 1, 4, 2):
             self.__restore_and_check_table(bundles[i], f'replica_view_{i}')
 
-    def test_embedding_index(self, reset_db: None, clip_embed: pxt.Function) -> None:
+    @pytest.mark.parametrize('embedding_precision', ['fp16', 'fp32'])
+    def test_embedding_index(self, reset_db: None, clip_embed: pxt.Function, embedding_precision: str) -> None:
         skip_test_if_not_installed('transformers')  # needed for CLIP
 
         t = pxt.create_table('tbl', {'image': pxt.Image})
         images = get_image_files()[:10]
         t.insert({'image': image} for image in images)
-        t.add_embedding_index('image', embedding=clip_embed)
+        t.add_embedding_index('image', embedding=clip_embed, precision=embedding_precision)  # type: ignore[arg-type]
```

Comment:
Can the `type: ignore` be avoided by typing the `embedding_precision` test parameter as `Literal[...]`?